session_1_exercise.py

```diff
@@ -0,0 +1,36 @@
+import random
+
+def drawBoard(board):
```

Comment:
nit: (for all these defs) I personally prefer snake case instead of camelcase for python (`drawBoard`->`draw_board`, `inputPlayerLetter`->`input_player_lettr`)